man/systemctl.xml

```diff
@@ -1178,12 +1178,17 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
                   </row>
                   <row>
                     <entry><literal>disabled</literal></entry>
-                    <entry>Unit file is not enabled, but contains an <literal>[Install]</literal> section with installation instructions.</entry>
+                    <entry>The unit file is not enabled, but contains an <literal>[Install]</literal> section with installation instructions.</entry>
                     <entry>&gt; 0</entry>
                   </row>
                   <row>
+                    <entry><literal>generated</literal></entry>
+                    <entry>The unit file was generated dynamically via a generator tool. See <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</entry>
+                    <entry>0</entry>
```

Comment:
Now that I read the next commit I see that generated files are not allowed to be enabled. This makes sense.
But still, I think some more explanation should be given how the state is assigned. First, mention that generated units cannot be enabled, than say that "States in the table are ordered by priority, and the first one that matches is returned.". And then sort the table appropriately. ?